src/network-analytics/README.md

```diff
@@ -0,0 +1,5 @@
+# Azure CLI NetworkAnalytics Extension #
+This is an extension to Azure CLI to manage NetworkAnalytics resources.
+
+## How to use ##
+Please add commands usage here.
```

Comment:
Please complete it with several examples.

Reply:
Added examples for all the commands.